src/index.html

```diff
@@ -11,6 +11,54 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+
+    <div class="searchBarBig">
+      <form 
+        class="inputDefault form formBig" 
+        action="/" 
+        method="get"
+        data-qa="big"
+        >
```

Comment:
Next time i will reject immediately when I see this
```suggestion
      <form 
        class="inputDefault form formBig" 
        action="/" 
        method="get"
        data-qa="big"
      >
```